src/java/org/apache/cassandra/index/sai/SSTableContextManager.java

```diff
@@ -52,7 +52,7 @@ public class SSTableContextManager
      * @return a set of contexts for SSTables with valid per-SSTable components, and a set of
      * SSTables with invalid or missing components
      */
-    public Pair<Set<SSTableContext>, Set<SSTableReader>> update(Collection<SSTableReader> removed, Iterable<SSTableReader> added, boolean validate)
+    public Pair<Set<SSTableContext>, Set<SSTableReader>> update(Collection<SSTableReader> removed, Iterable<SSTableReader> added, IndexValidation validate)
```

Comment:
I'd call the parameter `validation`. Also, the JavaDoc `@param` above is outdated.

Reply:
fixed